vulcanai/models/basenetwork.py

```diff
@@ -651,9 +652,11 @@ def fit(self, train_loader, val_loader, epochs,
                 else:
                     save_path = save_path + '/' + self.name + '_'
                 save_path = get_save_path(save_path, vis_type='train')
-            iterator = trange(epochs, desc='Epoch: ')
+            # iterator = trange(epochs, desc='Epoch: ')
 
-            for epoch in iterator:
+            for epoch in range(epochs):
+
+                logger.info('\n -------- Epoch: {}  --------\n'.format(epoch))
```

Comment:
why have this additional epoch log if the tqdm writer also writes the current epoch?

Reply:
Because I don't use tqdm on the epoch level anymore

Reply:
See my comment on the next post

Reply:
please see my other comment about this. I don't think the solution for this is to remove the progress bar altogether. unless I'm missing something, we just need to clean up the nested progress bar implementation.